rpms/systemd/sources/FB--add-back-compat-libs.patch

```diff
@@ -79,8 +79,9 @@ index 0c27f81..40138b2 100644
 +	-Wl,--version-script=$(top_srcdir)/src/compat-libs/libsystemd-journal.sym
 +
 +libsystemd_journal_la_LIBADD = \
-+	$(libsystemd_internal_la_LIBADD) \
-+	$(libsystemd_journal_internal_la_LIBADD)
++	libsystemd-journal-internal.la \
```

Comment:
should lines 82 and 83 still be the variables and we just add libbasic.so? (Since it's a patch of patch I don't have the full source at my fingertips...)